ci/build.sh

```diff
@@ -51,16 +51,26 @@ cd ../..
 echo "Test docker image locally..."
 ruby deploy/test/test_docker.rb
 
+# Set up Github
+if [ -n "$GITHUB_TOKEN" ]; then
+  git config --global user.email "travis@travis-ci.org"
+  git config --global user.name "Travis CI"
+  git remote add origin-repo https://${GITHUB_TOKEN}@github.com/SumoLogic/sumologic-kubernetes-collection.git > /dev/null 2>&1
+fi
+
 if [ "$TRAVIS_BRANCH" != "master" ] && [ "$TRAVIS_EVENT_TYPE" == "push" ] && [ -n "$GITHUB_TOKEN" ]; then
   echo "Generating yaml from helm chart..."
   echo "# This file is auto-generated." > deploy/kubernetes/fluentd-sumologic.yaml.tmpl
-  sudo helm template deploy/helm/sumologic --namespace "\$NAMESPACE" --set dryRun=true >> deploy/kubernetes/fluentd-sumologic.yaml.tmpl
+  sudo helm init --client-only
+  cd deploy/helm/sumologic
+  sudo helm dependency update
+  cd ../../../
+
+  with_files=`ls deploy/helm/sumologic/templates/*.yaml | sed 's#deploy/helm/sumologic/templates#-x templates#g' | sed 's/yaml/yaml \\\/g'`
```

Comment:
I tried running this command and I got `sed: 1: "s/yaml/yaml \\\/g": unterminated substitute in regular expression`, are we supposed to have only two `\` rather than three?

Reply:
yeah somehow when I tested it directly in my terminal `\\/g` worked but not in the bash script

Reply:
Weird, thanks for clarifying 👍